frontend/src/app/testResults/submissions/Submissions.tsx

```diff
@@ -0,0 +1,176 @@
+import { useParams } from "react-router-dom";
+import moment from "moment";
+import { DatePicker, Label } from "@trussworks/react-uswds";
+import { useState } from "react";
+
+import Pagination from "../../commonComponents/Pagination";
+import { formatDateWithTimeOption } from "../../utils/date";
+import {
+  GetUploadSubmissionsQuery,
+  useGetUploadSubmissionsQuery,
+} from "../../../generated/graphql";
+import {
+  // setEndTimeForDateRange,
```

Comment:
obsolete comment?

Reply:
nice catch! I used it and then decided not to use it